src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@englishman yes, true, there is an example in the book. And it's wrong. I will go through the book and try to find similar examples and will fix them. Thanks for catching that. We must not do anything inside ctors, except assigning private attributes. It's important.